client/src/components/GameLogs.vue

```diff
@@ -77,6 +77,9 @@
       },
       async getAllGames () {
         const response = await GameLogsService.fetchGames()
+        response.data.gamelogs.sort(function (a, b) {
+          return (new Date(b.date)) - (new Date(a.date))
```

Comment:
Can we standardize moment across the project since Vue date doesn't have as many features?

Reply:
Yes, that looks like the way to go.